guide/private-key-management/manual-backup.md

```diff
@@ -35,18 +35,18 @@ https://www.figma.com/community/file/995256542920917246/BDG---Private-key-manage
 
 # Manual backup / Recovery phrase
 
-Manual backup of the [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) has been the most common private key management scheme by far since its proposal with [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) in 2013. If you have used any [non-custodial]({{ '/guide/glossary/wallet/#non-custodial--custodial-wallet' | relative_url }}) bitcoin application you are likely to have experienced the onboarding requirements of manual backups.
+Manual backup of the [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) has been the most common private key management scheme by far since its proposal with [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) in 2013. If you have used any [non-custodial]({{ '/guide/glossary/wallet/#non-custodial--custodial-wallet' | relative_url }}) Bitcoin application, you are likely to have experienced the onboarding requirements of manual backups.
```

Comment:
```suggestion
Manual backup of the [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) has been the most common private key management scheme since its proposal with [BIP39](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki) in 2013. If you have used any [non-custodial]({{ '/guide/glossary/wallet/#non-custodial--custodial-wallet' | relative_url }}) Bitcoin application, you are likely to have experienced the onboarding requirements of manual backups.
```

Reply:
I think manual backups have been the most common by far. There is a difference between being "most common" (say, 51%) and being "by far most common" (say, 75-90%). I have no data to backup the claim, just going off what I have experienced. Thoughts anybody?